monkey/infection_monkey/exploit/zerologon_utils/remote_shell.py

```diff
@@ -70,10 +70,7 @@ def __init__(self, share, win32Process, smbConnection, outputFilename, secrets_d
         self.__noOutput = False
         self.__secrets_dir = secrets_dir
 
-        # We don't wanna deal with timeouts from now on.
-        # TODO are we sure we don't need timeout anymore?
         if self.__transferClient is not None:
-            self.__transferClient.setTimeout(100000)
```

Comment:
See https://github.com/guardicore/monkey/pull/1773#discussion_r824456432.
We need some kind of timeout for this.

Reply:
I don't think we need it because we have timeout set for the `smbConnection` before this.